addons/website/static/src/scss/website.scss

```diff
@@ -1458,3 +1458,15 @@ $ribbon-padding: 100px;
     // hide them if a modal is open.
     display: none !important;
 }
+
+// web/bootstrap_review.scss makes the card's background white
+// no matter what. We need to adjust theme colors for that
+.o_portal main {
+    table, .card, .navbar, .breadcrumb {
+        a, table a {
+            &:not(.btn) {
+                color: increase-contrast(o-color('o-cc1-link'), $card-bg);
+            }
```

Comment:
wouldn't that be enough ?
```
        a:not(.btn), .btn-link {
             color: increase-contrast(o-color('o-cc1-link'), $card-bg);
        }
```

Reply:
It's seems not specific to the portal module. 
Don't we need to fix this everywhere ?
+ I think that you also need to adapt the hover color..